.gitignore

```
@@ -1,4 +1,4 @@
 node_modules
 *.code-workspace
-dist
+build
```

I think `dist` fits better
```suggestion
dist
```